packages/config/config/devices/0x027a/zen04.json

```
@@ -0,0 +1,114 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN04",
+	"description": "Indoor Smart Plug",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xb002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "LED Indicator Mode",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Follow outlet",
+					"value": 0
+				},
+				{
+					"label": "Opposite of outlet",
+					"value": 1
+				},
+				{
+					"label": "Always off",
+					"value": 2
+				},
+				{
+					"label": "Always on",
+					"value": 3
+				}
+			]
```

Please change to this for consistency:
```suggestion
		"label": "LED Indicator",
		"valueSize": 1,
		"defaultValue": 0,
		"allowManualEntry": false,
		"options": [
			{
				"label": "On when load is on",
				"value": 0
			},
			{
				"label": "On when load is off",
				"value": 1
			},
			{
				"label": "Always off",
				"value": 2
			},
			{
				"label": "Always on",
				"value": 3
			}
		]
```

---

(looks like I messed up the indentation, but the bot will correct that)